src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,22 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private final UserService userService = new UserService();
+
     /**
-     * Imagine that some user wants to login to your site.
+     * Imagine that some user wants to log to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
```

Comment:
there is no need to change these lines

Reply:
maven wouldn't let me through and displayed an error. I had to change 'login' to 'log'